web/react/components/user_settings/custom_theme_chooser.jsx

```diff
@@ -19,6 +19,15 @@ export default class CustomThemeChooser extends React.Component {
         });
         $('.color-picker').on('changeColor', this.onPickerChange);
     }
+    componentDidUpdate() {
```

Comment:
I am confused. Why is this in componentDidUpdate and not when something is pasted in the theme box?